CONTRIBUTING.md

```diff
@@ -93,7 +93,8 @@ Then, run `bin/setup` to install Ruby and Node dependencies and set up the
 database.
 
 Once setup completes, configure your local development environment.
-1. Open `.env` and make any changes.
+1. Copy `.env.sample` to `.env`
```

Comment:
The `bin/setup` script should copy the file from `.env.sample` to `.env` but it's reasonable to break it into two steps.

Reply:
Ohh it's possible I didn't run the bin/setup step yet when I noticed this.

Reply:
Ahhh maybe we're doing it in both places for some silly reason.

Reply:
Doesn't `bin/setup` call `bin/setup-rails` at some point? Is the line in both scripts?

Reply:
(Do these need to be separate scripts at all? Does anyone use the standalone script?)

Reply:
I think it's because we renamed the `bin/setup` that comes with `rails` to an independent script so we could write our own script for installing other dependencies... but we could inline them all into a ruby script!

Reply:
To answer my own questions:
* yes, the copying is in both `bin/setup` and `bin/setup-rails`
* one of the scripts in `bash` and the other `ruby`, which makes combining them more work than just copy-pasting one into the other, which to me makes combining the scripts not worth it (but i'm happy for someone who is excited to do it to go for it)

So I'd say, if the spirit moves you, delete lines 41-46 in `bin/setup` to remove the redundant copying (and just assuming that we want to keep as much of the logic as possible in the Ruby script, because it seems nicer than bash). But it doesn't seem a big deal either way.

:shipit: